Flutter/cartesian-charts/zoom-pan.md

```diff
@@ -327,6 +335,11 @@ If zoom mode is set to [`zoomMode.x`](https://pub.dev/documentation/syncfusion_f
 
 Also refer [`zooming`](./callbacks#onzooming), [`zoom start`](./callbacks#onzoomstart) and [`zoom end`](./callbacks#onzoomend) events for customizing the zooming further.
 
+### Calculate range based on visible points
+
+[`anchorRangeToVisiblePoints`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartAxis/anchorRangeToVisiblePoints.html) property can be set to `false` the value axis range will be calculated based on all the data points in the chart irrespective of visible points. For further reference, check this [AutoRangeCalculation](./axis-customization.md) section.
+
```

Comment:
why the gif image for anchorRangeToVisiblePoints exists? already conveyed to remove this

Reply:
have removed the gif image

Reply:
yes have removed